### PR TITLE
chore: add changeset

### PR DESCRIPTION
**Pull request by sashuk**
[FX-3931]

### Description

This pull request prepares the changeset message that will summarise all the breaking changes made as part of the https://toptal-core.atlassian.net/browse/FX-3859 epic.

⚠️ Please review it as if you have no context about the breaking changes.

🐛 Failing visual tests are addressed in https://github.com/toptal/picasso/pull/3541

### How to test

- Review the https://github.com/toptal/picasso/blob/FX-3931-add-breakpoints-changeset/.changeset/green-countries-table.md changeset

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [N/A] Make sure that additions and changes on design follow [Toptal's BASE design](https://design.toptal.net/), and it's been already discussed with designers at #-base-core
- [N/A] Annotate all `props` in component with documentation
- [N/A] Create `examples` for component
- [N/A] Ensure that deployed demo has expected results and good examples
- [N/A] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- [N/A] Covered with tests

**Breaking change**

- [N/A] codemod is created and showcased in the changeset
- [ ] test alpha package of Picasso in StaffPortal

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-3931]: https://toptal-core.atlassian.net/browse/FX-3931?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by sashuk** [comment]
@toptal-anvil ping reviewers

File: .changeset/green-countries-table.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  - before: `screenSizeToBreakpointKey(300) = 'small`; now: `screenSizeToBreakpointKey(300) = 'xs` (because `300px` is between `xs` and `sm` breakpoints)
+  - before: `screenSizeToBreakpointKey(2000) = 'extra-large`; now: `screenSizeToBreakpointKey(2000) = 'xl` (because `2000px` is bigger than the biggest `xl` breakpoint)
+- `isScreenSize()`, `useBreakpoint()`, and `useScreens()` hooks also changed their behavior due to the same changes in breakpoint ranges.
+
+### Grid ([#3538](https://github.com/toptal/picasso/pull/3538))
+
+- in order to comply with BASE design, `Grid` spacing property automatically adjusts to the screen size, unless explicitly specified by consumer. Extra-small and small screens have `16px` spacing, medium screens have `24px` spacing and large and extra-large screens have `32px` spacing. If you want to keep the old behavior, please explicitly set `spacing={32}` for `Grid` components.
```
Comment (OleksandrNechai) [inline .changeset/green-countries-table.md:33]: I am not sure, but outside of Picasso people call this gutter? If so, maybe makes sense to mention it here.

**Review by OleksandrNechai — APPROVED** [review]
👍 good description. Concise and clear.

File: .changeset/green-countries-table.md (hunk 1 above)
Comment (sashuk) [reply .changeset/green-countries-table.md:33]: @OleksandrNechai Agree, `gutters` were replaced with `spacing` (as spacing was already mentioned before), thanks!

**Review by mkrl — APPROVED** [review]
Love the description and migration guide, nice job!

File: .changeset/green-countries-table.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+Before:
+Value         |0px     576px    768px    992px    1920px
+Breakpoint    |xs      sm       md       lg       xl
+Screen width  |--------|--------|--------|--------|-------->
+Range         |   sm   |   md   |   lg   |   xl   |   xl
+After:
+Value         |0px     480px    768px    1024px   1440px
+Breakpoint    |xs      sm       md       lg       xl
+Screen width  |--------|--------|--------|--------|-------->
+Range         |   xs   |   sm   |   md   |   lg   |   xl
```
Comment (augustobmoura) [inline .changeset/green-countries-table.md:21]: This looks great!

**Review by augustobmoura — APPROVED** [review]

**Pull request merged**